mindee/documents/financial/line_item.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional
+
+from mindee.fields.base import FieldPositionMixin, TypePrediction, float_to_string
+
+
+class InvoiceLineItem(FieldPositionMixin):
```

Comment:
Instead of copy paste it all the properties, we could add a inheritance from the Invoice V4 class.
It is a pragmatic solution to avoid copy pasting but using its own class in case of future evolution.

Reply:
inherit directly, we can always change later